pwiz_tools/Skyline/Skyline.cs

```diff
@@ -3321,6 +3325,19 @@ private void ShowPeptideSettingsUI(IWin32Window parent, PeptideSettingsUI.TABS?
                     if (oldStandard != newStandard)
                         AddStandardsToDocument(newStandard, missingPeptides);
                 }
+

```

Comment:
And this should go in something like OkDialog() or OnClosing() for the form. SkylineWindow is such a crazy long class already. Hate seeing logic like this leak out into it.

Reply:
Done.